src/Task/Phpcs.php

```diff
@@ -46,7 +46,7 @@ public function getConfigurableOptions()
             'report_width' => null,
         ]);
 
-        $resolver->addAllowedTypes('standard', ['null', 'string']);
+        $resolver->addAllowedTypes('standard', ['array']);
```

Comment:
Can you allow both array and string and cast a string to an array? That way, old configuration doesnt break.

Reply:
Yeah, that was my first attempt at this, but it wasn't working as validation would fail .

Reply:
I will try to make it work tomorrow..

Reply:
Hi @veewee,

Turns out it actually possible - so the configuration is now backward compatible! Win. 
Thanks for bringing it up!

I've done a forced push so you may want to review it again.